### ISSUE TITLE
SpaceNotSupported on continuousMove.

### ISSUE THREAD

**Issue by mbullington** [agsh/onvif#9]
Hello, I'm just trying to get continuousMove to work with my ONVIF camera. It supports Pan and Tilt, but not Zoom. When I try to use continuousMove with ```{x: 1}```, I get this error as a result. The ONVIF specification says that means ```A space is referenced in an argument which is not supported by the PTZ
node.``` Thank you for this library.

```
Error: ONVIF SOAP Fault: {
  "value":"SOAP-ENV:Sender",
  "subcode": {
    "value":"ter:InvalidArgVal",
    "subcode": {
      "value":"ter:SpaceNotSupported"
    }
  }
}
```

**Comment by agsh** [comment]
Hi!
First of all please check your PTZ configuration options. Does your camera supports such space (1 - is a very high value)? To do this:
```js
cam.getConfigurations(function(err, configurations) {
  // configurations - is an object with all configurations where keys are configuration's tokens
  console.log(configurations);
  // let's check first conf
  cam.getConfigurationOptions(Object.keys(configurations)[0], function(err, options) {
    console.log(options.spaces); // here is spaces for all movements
  }
});
```
Also in code I forcibly set all undefined parameters (x, y, zoom) to 0. Maybe this causes an error.
I will be waiting for you results.

**Comment by agsh** [comment]
Closed with 2757da4b0e5a22f63a52e64aaef6e5a344e45215
